src/server/server.ts

```diff
@@ -151,6 +193,10 @@ export default class Server implements Hub.RouteBuilder {
     return this.absUrl(`/actions/${encodeURIComponent(action.name)}/form`)
   }
 
+  private oauthRedirectUrl(action: Hub.Action) {
+    const url = this.absUrl(`/actions/${encodeURIComponent(action.name)}/oauth_redirect`)
+    return url
```

Comment:
Just return the call to `this.absUrl` unless you're expecting to do something else with it.

Reply:
Can't hurt to add return types (`string`) to both of the functions too

Reply:
the return type should be inferred